test/eth/Cdp.spec.js

```diff
@@ -296,6 +325,39 @@ const sharedTests = openCdp => {
             );
           }
         });
+
+        test('wipe debt with non-zero stability fee', async () => {
```

Comment:
why was it necessary to move these three tests into the "with drip" block?

Reply:
Because of the order these tests run, they were shutting the cdp in the `with debt` block and then going on to try to use it again in `with drip`. So we could also solve this problem by opening, locking, and drawing in the `beforeAll` in `with drip`, but this seemed more efficient